add note to reference AWS secrets for apikey (#7721)
DataDog/integrations-core: ecs_fargate/README.md

              enable-ecs-log-metadata: true
          MemoryReservation: 50
```

> **Note**: You can also use a [TaskDefinition secret][37] if you do not want to expose the apiKey as a plain-text variable.

Reviewer: ```suggestion
**Note**: Use a [TaskDefinition secret][37] to avoid exposing the `apikey` in plain text.

```
Need a new line or else Markdown sticks the next sentence into the same line as the Note.
